metaflow/util.py

```diff
@@ -167,14 +167,7 @@ def get_username():
     could not be determined.
     """
     # note: the order of the list matters
-<<<<<<< HEAD
-    if METAFLOW_USER:
-        return METAFLOW_USER
```

Comment:
You missed this! `METAFLOW_USER` added in https://github.com/zillow/metaflow/pull/12

Reply:
Ah. I was thinking why this was needed and myself in https://github.com/zillow/metaflow/pull/12/files#r490528824 answered it. Fixing...

Reply:
Looks like something that can be contributed back to `netflix/metaflow`